src/output.css

```diff
@@ -556,15 +556,72 @@ video {
   --tw-contain-style:  ;
 }
 
-.font-outfit{
-  font-family: Outfit, sans-serif;
+.absolute{
+  position: absolute;
+}
+
+.relative{
+  position: relative;
+}
+
+.left-1\/2{
+  left: 50%;
+}
+
+.top-1\/2{
+  top: 50%;
+}
+
+.z-10{
+  z-index: 10;
+}
+
+.flex{
```

Comment:
This is also related to the comment mentioned above. You can use TailwindCSS, there's no need of defining this class.